File: app/src/main/java/com/suhane/flickr/ui/photos/PhotoViewListener.java
package com.suhane.flickr.ui.photos;

import com.suhane.flickr.repository.model.photos.search.Photo;

/**
 * Created by shashanksuhane on 11/02/18.
 */

public interface PhotoViewListener {
    void onLoadMore();
    void onItemClick(Photo photo);
}

File: app/src/main/java/com/suhane/flickr/ui/photos/PhotoViewListener.kt
package com.suhane.flickr.ui.photos

import com.suhane.flickr.repository.model.photos.search.Photo

/**
 * Created by shashanksuhane on 11/02/18.
 */

interface PhotoViewListener {
    fun onLoadMore()
    fun onItemClick(photo: Photo)
}
